Dockerfile

```diff
@@ -16,6 +17,8 @@ RUN apk add --no-cache --virtual .build-dependencies build-base curl && \
     apk del .build-dependencies ocaml && \
     rm -rf /tmp/unison-${UNISON_VERSION}
 
+RUN apk add --no-cache su-exec
```

Comment:
This should go on line 10 together with `shadow` (i.e. `apk add --no-cache shadow su-exec`)

Reply:
I don't have a problem moving it, but I'm curious why you want it moved.  Is there any reason besides just consolidating invocations to apk?

The reason I didn't put it there is because it isn't required for the build, it's just required for runtime.  So I thought this fact is more evident if I add the package after the build is complete.

Please confirm you still want it moved.

Reply:
Every RUN creates a new layer of a Docker image.